plugins/module_utils/foreman_helper.py

```diff
@@ -1060,9 +1022,8 @@ def _foreman_spec_helper(spec):
     #
     # So in conclusion, we only have to verify that apypie is available before using it.
     # Lazy evaluation helps there.
-    inflector = HAS_APYPIE and apypie.Inflector()
     for key, value in spec.items():
-        foreman_value = {k: v for (k, v) in value.items() if k in ['type', 'ensure']}
+        foreman_value = {k: v for (k, v) in value.items() if k in ['ensure', 'type']}
```

Comment:
Should `['ensure', 'type']` be in some `_STATIC`?

Reply:
It is now.